pkg/controllers/dnspolicy/dnspolicy_dnsrecords.go

```diff
@@ -174,7 +186,7 @@ func getClusterGatewayAddresses(gw *gatewayv1beta1.Gateway) map[string][]gateway
 		var gatewayAddresses []gatewayv1beta1.GatewayAddress
 
 		//Default to Single Cluster (Normal Gateway Status)
-		cluster := "none"
+		cluster := singleCluster
 		addressValue := address.Value
```

Comment:
I think the changes here are fine as a temp solution until we get https://github.com/Kuadrant/multicluster-gateway-controller/issues/496, which will hopefully remove the dependency on the name.

Although it is still possible for someone to mess this up, i guess it is less likely they will call there cluster "kudarant.io/single"